### ISSUE TITLE
Doesn't work with Zabbix 6.x

### ISSUE THREAD

**Issue by danboid** [hermanekt/zabbix_systemd#5]
Should your template work with Zabbix 6.0? It doesn't seem to be working for me.

I am running zabbix agent2 on my client and I was able to import Template_Systemd.xml OK but when I look at "Latest data" for the machine I'm monitoring I don't see any systemd related data. Both the Zabbix server and the client are running Ubuntu 22.04.

I want a template to monitor when specific systemd services stop running such as ssh, apache2 and mysql etc. I've not been able to get the official Zabbix systemd template to work properly.

**Comment by hermanekt** [comment]
Hi thank you for your question. Are you sure you are using Zabbix Agent 2 on your monitoried machine?

**Comment by danboid** [comment]
Yes. No worries tho as I worked out how to use the official systemd template.

Thanks
